contrib/devtools/copyright_header.py

```diff
@@ -65,7 +65,7 @@ def get_filenames_to_examine():
 ################################################################################
 
 
-COPYRIGHT_WITH_C = 'Copyright \(c\)'
+COPYRIGHT_WITH_C = 'Copyright\s*\(c\)'
```

Comment:
As @pep8speaks suggests, you might want to use raw string literals here: `r'Copyright\s*\(c\)'`

Reply:
Will do.

Reply:
be careful with that, pep8speaks is wrong here because it doesn't know that this string will be used in some regular expressions.